lib/xcodeproj/constants.rb

```diff
@@ -138,13 +138,13 @@ module Constants
         'MTL_ENABLE_DEBUG_INFO'             => 'NO',
       }.freeze,
       [:ios] => {
-        'SDKROOT'                           => 'iphoneos',
+        'SDKROOT'                           => `xcrun xcode-select --print-path`.strip + "/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS" + LAST_KNOWN_IOS_SDK + ".sdk",
```

Comment:
These would be absolute paths and therefore make this fragile and won't allow checking in Pods directory as there will be hard-coded paths specific to Xcode installs.